dev/com.ibm.ws.jaxrs.2.1.common/bnd.bnd

```diff
@@ -76,12 +74,11 @@ Include-Resource:\
  META-INF/services/javax.ws.rs.client.ClientBuilder=resources/META-INF/services/javax.ws.rs.client.ClientBuilder,\
  META-INF/services/javax.ws.rs.ext.RuntimeDelegate=resources/META-INF/services/javax.ws.rs.ext.RuntimeDelegate,\
  META-INF/services/javax.ws.rs.sse.SseEventSource$Builder=resources/META-INF/services/javax.ws.rs.sse.SseEventSource$Builder
- 
+
 
 -dsannotations: com.ibm.ws.jaxrs20.component.*, \
   com.ibm.ws.jaxrs20.clientconfig.JAXRSClientConfigImpl, \
-  com.ibm.ws.jaxrs20.providers.customexceptionmapper.CustomExceptionMapperRegister, \
-  com.ibm.ws.jaxrs20.providers.security.SecurityAnnoProviderRegister
+  com.ibm.ws.jaxrs20.providers.customexceptionmapper.CustomExceptionMapperRegister
 
 instrument.classesExcludes: com/ibm/ws/jaxrs20/internal/resources/*.class
 
```

Comment:
The buildpath changes should be undone that were done previously.  If com.ibm.ws.security.authorization.util can be removed that would be good as well since then it make sense that it can be removed from the common jaxrs2.1 feature files.

Reply:
So can you remove com.ibm.ws.security.authorization.util from the buildpath?  If you can't then you will need to update the features file.

Reply:
yes